### DISCUSSION TITLE
web UI keeps stuck at taking image

### DISCUSSION THREAD

**Discussion by aimnfrs** [jomjol/AI-on-the-edge-device#2825] [Q&A]
Hi does anyone know how to resolve if my web UI keeps stuck at taking image? I have waited for so long
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/157106025/7f194598-c367-4182-aade-584c0d50de4e)
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/157106025/72398115-4057-4f3d-84d3-1eda3c71f1e8)

**Comment by caco3** [comment]
Please provide a debug log.
And make sure you have selected an existing model.
Maybe simply format and set the SD-Card up one more.
